### PR TITLE
[new release] sel (0.2.0)

### PR DESCRIPTION
**Pull request by gares**
Simple Event Library

- Project page: <a href="https://github.com/gares/sel">https://github.com/gares/sel</a>

##### CHANGES:

- Major API cleanup and documentation

### CODE REVIEW

File: packages/sel/sel.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+synopsis: "Simple Event Library"
+description: "Some sugar atop Unix.select"
+maintainer: ["Enrico Tassi"]
+authors: ["Enrico Tassi"]
+license: "MIT"
+tags: ["event" "input" "output"]
+homepage: "https://github.com/gares/sel"
+bug-reports: "https://github.com/gares/sel/issues"
+depends: [
+  "ocaml"
```
Comment (mseri) [inline packages/sel/sel.0.2.0/opam:11]:
```suggestion
  "ocaml" {>= "4.07"}
```
Due to the use of `Stdlib`
```

#=== ERROR while compiling sel.0.2.0 ==========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.03.0 | pinned(https://github.com/gares/sel/releases/download/v0.2.0/sel-0.2.0.tbz)
# path                 ~/.opam/4.03/.opam-switch/build/sel.0.2.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p sel -j 71 @install
# exit-code            1
# env-file             ~/.opam/log/sel-7-6eacdd.env
# output-file          ~/.opam/log/sel-7-6eacdd.out
### output ###
# (cd _build/default && /home/opam/.opam/4.03/bin/ocamlopt.opt -w -40 -g -I lib/.sel.objs/byte -I lib/.sel.objs/native -I /home/opam/.opam/4.03/lib/ppx_deriving/runtime -I /home/opam/.opam/4.03/lib/result -intf-suffix .ml -no-alias-deps -o lib/.sel.objs/native/sel.cmx -c -impl lib/sel.pp.ml)
# File "lib/sel.ml", line 46, characters 27-48:
# Error: Unbound module Stdlib
# (cd _build/default && /home/opam/.opam/4.03/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/.sel.objs/byte -I /home/opam/.opam/4.03/lib/ppx_deriving/runtime -I /home/opam/.opam/4.03/lib/result -intf-suffix .ml -no-alias-deps -o lib/.sel.objs/byte/sel.cmo -c -impl lib/sel.pp.ml)
# File "lib/sel.ml", line 46, characters 27-48:
# Error: Unbound module Stdlib
```

**Comment by gares** [comment]
Thanks

File: packages/sel/sel.0.2.0/opam [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+synopsis: "Simple Event Library"
+description: "Some sugar atop Unix.select"
+maintainer: ["Enrico Tassi"]
+authors: ["Enrico Tassi"]
+license: "MIT"
+tags: ["event" "input" "output"]
+homepage: "https://github.com/gares/sel"
+bug-reports: "https://github.com/gares/sel/issues"
+depends: [
+  "ocaml" {>= "4.07"}
```
Comment (avsm) [inline packages/sel/sel.0.2.0/opam:11]:
```suggestion
  "ocaml" {>= "4.08"}
```

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Pull request merged**